### PR TITLE
Adds support for list type inputs

### PR DESCRIPTION
**Pull request by ibraheem-opentensor**
- Users can now use `--netuid 1,2,3,4` or `--netuid "1, 2, 3, 4"` instead of `-n 1 -n 2 -n 3`
- Extended to netuids, weights, salt
- Explicitly ask for hotkey name when creating a new hotkey
- Support for both ss58 and wallet name in include,exclude hotkeys
- list convention extendted to include and exclude hotkeys
- Exit early in-case no subnets found in sudo set (which is why added return in _run command)
- Fixes balance conversion in stake multiple

### CODE REVIEW

File: bittensor_cli/cli.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ class Options:
         show_default=False,
     )
     netuids = typer.Option(
-        [],
+        None,
         "--netuids",
         "-n",
         help="Set the netuid(s) to filter by. Separate multiple netuids with a space, for example: `0 1 2`.",
```
Comment (thewhaleking) [inline bittensor_cli/cli.py:144]: Needs updated

File: bittensor_cli/cli.py [hunk 2]
```diff
@@ -149,7 +149,7 @@ class Options:
         prompt=True,
     )
     weights = typer.Option(
-        [],
+        None,
         "--weights",
         "-w",
         help="Weights for the specified UIDs, e.g. `-w 0.2 -w 0.4 -w 0.1 ...` Must correspond to the order of the UIDs.",
```
Comment (thewhaleking) [inline bittensor_cli/cli.py:155]: Needs updated

**Review by thewhaleking — CHANGES_REQUESTED** [review]

**Review by thewhaleking — APPROVED** [review]

**Comment by thewhaleking** [comment]
Approved, but do we want to do the same thing for `--include-hotkeys`, `--exclude-hotkeys`, and the like?

**Review by thewhaleking — APPROVED** [review]

**Comment by ibraheem-opentensor** [comment]
> Approved, but do we want to do the same thing for `--include-hotkeys`, `--exclude-hotkeys`, and the like?

Great call - let me implement this there too; then we can merge in one go

**Review by thewhaleking — APPROVED** [review]

**Pull request merged**